### PR TITLE
The settle_timeout range is inclusive

### PR DESCRIPTION
**Pull request by hackaugusto**
## Description

This fixes a mismatch among the proxies and the smart contract. The smart contract allowed inclusive values, while the proxies used exclusive checks.

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/network/proxies/token_network.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def new_netting_channel(
 
         timeout_min = self.settlement_timeout_min()
         timeout_max = self.settlement_timeout_max()
-        invalid_timeout = settle_timeout < timeout_min or settle_timeout > timeout_max
```
Comment (karlb) [inline raiden/network/proxies/token_network.py]: Going by the variable names, min and max should be allowed. Is there a reason against changing this in the contracts?

Comment (karlb) [reply raiden/network/proxies/token_network.py]: Actually, I don't think any change is needed. The contract checks for validity inclusively:
https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/data/source/raiden/TokenNetwork.sol#L198-L202

The Raiden code base checks for invalidity exclusively. This matches up perfectly.

Comment (karlb) [reply raiden/network/proxies/token_network.py]: I try to prove my point in https://github.com/raiden-network/raiden/pull/4774.

Comment (hackaugusto) [reply raiden/network/proxies/token_network.py]: Oh, you're right. My mistake.

**Pull request closed without merging**